defaults/main.yml

```diff
@@ -4,3 +4,11 @@ aws_cwa_disk_monitor_paths:
   - "/"
 
 aws_cwa_cfgs: generic-amazon-cloudwatch-agent.json.j2
+aws_cwa_url: "https://s3.amazonaws.com/amazoncloudwatch-agent/linux/amd64/latest/AmazonCloudWatchAgent.zip"
+aws_cwa_local_user: root # The user the CloudWatch Agent runs as.
+aws_cwa_user_path: "/"
```

Comment:
Should this not be
```suggestion
aws_cwa_user_path: "/root"
```

Reply:
Yup, already fixed but forgot to push last night's changes 🙄